internal/service/project/resource_project.go

```diff
@@ -372,5 +393,27 @@ func setProjectTerraformProperties(d *schema.ResourceData, client *aiven.Client,
 		return diag.FromErr(err)
 	}
 
+	var tags []map[string]interface{}
+	for k, v := range project.Tags {
+		tags = append(tags, map[string]interface{}{
+			"key":   k,
+			"value": v,
+		})
+	}
+	if err := d.Set("tag", tags); err != nil {
+		return diag.FromErr(err)
+	}
+
```

Comment:
Maybe this code should be extracted to a function? Then all properties in this function will have kind of the same set pattern. Something like the following:
```go
if err := setTags(d, project.Tags); err != nil {
    return diag.FromErr(err)
}
```

Reply:
done